change/@uifabric-charting-2020-03-26-19-19-23-user-v-satgar-PBI_4052037.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
+  "comment": "Add string inside Donut chart",
```

Comment:
change to more consistent release note comment structure "DonutChart: Add prop to allow string inside Donut."

Reply:
Changed as suggested